### PR TITLE
Refactor federation-up.sh script.

### PR DESCRIPTION
**Pull request by madhusudancs**
Move common variables to common.sh script in preparation for the upcoming changes to federation-down.sh that also depends on these common variables.

```release-note
NONE
```

cc @kubernetes/sig-federation-pr-reviews @shashidharatd

### CODE REVIEW

File: federation/cluster/common.sh [hunk 1]
```diff
@@ -54,6 +54,15 @@ KUBE_BUILD_STAGE=${KUBE_BUILD_STAGE:-release-stage}
 
 source "${KUBE_ROOT}/cluster/common.sh"
 
+# kubefed configuration
+FEDERATION_KUBE_CONTEXT="${FEDERATION_KUBE_CONTEXT:-e2e-federation}"
+HOST_CLUSTER_CONTEXT="${FEDERATION_HOST_CLUSTER_CONTEXT:-${1}}"
+
+# kube-dns configuration.
+KUBEDNS_CONFIGMAP_NAME="kube-dns"
+KUBEDNS_CONFIGMAP_NAMESPACE = "kube-system"
```
Comment (shashidharatd) [inline federation/cluster/common.sh]: i think you might need to remove spaces around assignment "=" operator.

**Review by madhusudancs — COMMENTED** [review]
@shashidharatd PTAL.

**Comment by shashidharatd** [comment]
The changes LGTM.

**Comment by shashidharatd** [comment]
@k8s-bot cri node e2e test this

**Comment by shashidharatd** [comment]
@k8s-bot cri gce node e2e test this

**Pull request merged**